modules/ding_provider/ding_provider.module

```diff
@@ -587,11 +590,16 @@ function ding_provider_get_pseudo_entity($ding_entity_id) {
   $ting_entity->ding_entity_id = $ding_entity_id;
   $ting_entity->provider_id = $local_id;
   $ting_entity->ding_entity_type = 'ding_entity';
-  $ting_entity->reply = new TingClientObject();
+  // TODO BBS-SAL: Clean out the direct property sets when done.
+  $ting_entity->reply = new TingPseudoObject();
   $ting_entity->reply->id = $ding_entity_id;
+  $ting_entity->reply->setDingId($ding_entity_id);
```

Comment:
Differentiating between opensearch ids and local ids is based on the OpenSearch architecture where the search system (OpenSearch) and management system (DDELibra, FBS) are separate.

I have not given the idea of keeping `getLocalId()` in the OpenSearch implementation but I would think it lead to increasing complexity in the calling code which may need to check for the existance of this method.

Since the foundation of the system is Danish I would prefer to keep both ids in SAL and just let the value be the same for integrated systems such as Primo/Alpeh.